src/assets/sass/protocol/base/elements/_sections.scss

```diff
@@ -4,18 +4,42 @@
 
 @import '../../includes/lib';
 
+
 // This wrapper element sets the content width
 .mzp-l-content {
-    @include border-box;
     @include clearfix;
     margin: 0 auto;
     max-width: $content-max;
     min-width: $content-xs;
     padding: $layout-xs;
     position: relative;
 
-    &.mzp-t-narrow {
+    &.mzp-t-content-sm {
```

Comment:
My personal preference would be to put the default class styles before the modifier styles in the source order. (so move the two media queries on lines 45 and 49 up above this line).

Totally optional, personal, nit-pic and I realize you just used the existing source order when editing so definitely not a blocker.

Reply:
Well, the true default is without a modifier... but that then raises the question of whether we should stick to the standard of "medium" being the default and make a modifier for `.mzp-t-content-max`

Reply:
Oh I follow now, you mean the media queries at the end... yeah I should move those.